<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormulaUI</class>
 <widget class="QDialog" name="FormulaUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>楷体</family>
    <pointsize>12</pointsize>
    <weight>50</weight>
    <italic>false</italic>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 12pt &quot;楷体&quot;;</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>570</y>
     <width>161</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTableWidget" name="FormulaListUI">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>731</width>
     <height>291</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>731</width>
     <height>291</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>731</width>
     <height>291</height>
    </size>
   </property>
   <property name="font">
    <font>
     <family>楷体</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;楷体&quot;;</string>
   </property>
   <property name="selectionBehavior">
    <enum>QAbstractItemView::SelectRows</enum>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>100</number>
   </attribute>
   <attribute name="horizontalHeaderMinimumSectionSize">
    <number>18</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>          公          式      </string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>最大连对</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>最近连对</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="eleEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>510</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="FormulaEdit">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>510</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>510</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>元素个数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>510</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>公式个数</string>
   </property>
  </widget>
  <widget class="QPushButton" name="GenerateFormula">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>570</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>生成公式</string>
   </property>
  </widget>
  <widget class="QPushButton" name="predictBT">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>570</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>预测下期号码</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>540</y>
     <width>71</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>预测个数</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PredictNumberEdit">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>540</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTableWidget" name="LotterList">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>310</y>
     <width>731</width>
     <height>191</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>104</number>
   </attribute>
   <attribute name="verticalHeaderVisible">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>红球1</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>红球2</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>红球3</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>红球4</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>红球5</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>红球6</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>蓝球</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="clearLotter">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>570</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>清除所有</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="MaxContinue">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>540</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>540</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>最大连对&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="RecentContinue">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>540</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>540</y>
     <width>81</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>最近连对&gt;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FormulaUI</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FormulaUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
